pgpool2_exporter.yaml

```diff
@@ -0,0 +1,108 @@
+package:
+  name: pgpool2_exporter
+  version: "1.2.2"
+  epoch: 0
+  description: Prometheus exporter for Pgpool-II metrics.
+  copyright:
+    - license: MIT
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: http://github.com/pgpool/pgpool2_exporter
+      tag: v${{package.version}}
+      expected-commit: 06a68dead344308c3346f53aba811c929eff71d5
+
+  - uses: go/build
+    with:
+      packages: ./cmd/pgpool2_exporter
+      modroot: .
```

Comment:
I think this will need tags too? https://github.com/pgpool/pgpool2_exporter/blob/06a68dead344308c3346f53aba811c929eff71d5/.promu.yml#L10

Reply:
Since:
```bash
        build   -ldflags="-X github.com/prometheus/common/version.Version=1.2.2 -X github.com/prometheus/common/version.Revision=06a68dead344308c3346f53aba811c929eff71d5 -X github.com/prometheus/common/version.Branch=master -X github.com/prometheus/common/version.BuildUser=root@server2 -X github.com/prometheus/common/version.BuildDate=20240312-06:23:47  -extldflags '-static'"
        build   -tags="netgo static_build"
        build   CGO_ENABLED=0
        build   GOARCH=amd64
        build   GOOS=linux
 ```

Reply:
Thanks for pointing this, I've added tag instructions for build.